easybuild/easyblocks/b/bamtools.py

```diff
@@ -58,10 +58,27 @@ def sanity_check_step(self):
         sharedlib_ext = get_shared_lib_ext()
 
         custom_paths = {
-            'files': ["bin/bamtools", "include/shared/bamtools_global.h", "lib/libbamtools.a",
-                      "lib/libbamtools.%s" % sharedlib_ext, "lib/libbamtools-utils.%s" % sharedlib_ext,
-                      "lib/libjsoncpp.%s" % sharedlib_ext],
-            'dirs': ["include/api", "docs"]
+            'files': [
+                "bin/bamtools",
+                "include/shared/bamtools_global.h",
+                "lib/libbamtools.a",
+                "lib/libbamtools.%s" % sharedlib_ext
```

Comment:
I'm having second thoughts about this, it doesn't follow the style of other easyblocks...